src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,84 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+import React from 'react';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const pagesAmount = Math.ceil(total / perPage);
+  const pages: number[] = [];
```

Comment:
You don't need a loop here for creating an array 
```suggestion
  const pages: number[] = Array.from({ length: pagesAmount }, (_, index) => index + 1);
```